pinot-common/src/test/java/org/apache/pinot/common/function/AggregationFunctionTypeTest.java

```diff
@@ -18,10 +18,9 @@
  */
 package org.apache.pinot.common.function;
 
+import org.apache.pinot.segment.spi.AggregationFunctionType;
```

Comment:
Let's a move the tests?

Reply:
Ah yes, even though the intention was to only move the interfaces, some POJOs got moved as well. I'll move their tests to the new move.

Reply:
Done.